runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/graph/DeduceFlattenLocationsFunction.java

```diff
@@ -156,8 +156,8 @@
 
     // Find all predecessor and successor locations for every flatten.
     for (Node flatten : flattens) {
-      AggregatedLocation predecessorLocations = AggregatedLocation.NEITHER;
-      AggregatedLocation successorLocations = AggregatedLocation.NEITHER;
+      AggregatedLocation predecessorLocations;
+      AggregatedLocation successorLocations;
```

Comment:
Please combine these two lines with the next two lines

Reply:
Done!